packages/blocks/github-pr-overview/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "github-pr-overview",
+  "version": "0.0.0",
+  "description": "github-pr-overview block",
+  "keywords": [
+    "blockprotocol",
+    "blocks",
+    "github",
+    "pullrequest"
+  ],
+  "license": "MIT",
+  "author": "Alfred Mountfield",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
+    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "block-scripts serve"
+  },
+  "dependencies": {
+    "@mui/icons-material": "5.4.2",
+    "@mui/lab": "5.0.0-alpha.71",
+    "@mui/material": "5.4.3",
+    "blockprotocol": "0.0.8",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.3"
+  },
+  "devDependencies": {
+    "block-scripts": "0.0.5",
+    "mock-block-dock": "0.0.7",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "block-scripts": {
+    "port": 63212
+  },
+  "blockprotocol": {
+    "displayName": "GithubPullRequestOverview",
+    "examples": [
+      {}
+    ],
+    "icon": "public/omega.svg",
+    "image": "public/block-preview.svg"
```

Comment:
What should we do for these three?

Definitely think I need to figure out how to include an example

Reply:
I've added an MIT-licensed icon for a PR. It's a bit tall and seems to get squished in the HASH UI but we can worry about that later I reckon.

Not sure how to make a nice svg preview image. Perhaps it's not worth doing until we iterate on the block a bit more anyway

Reply:
I think the icon can most likely be quickly fixed by making it square in dimensions.

@MaggieAppleton will handle the preview image as for our other base/starter blocks (and I suggest we do indeed leave as-is for now).